frontend/src/pages/upload/UploadPage.tsx

```diff
@@ -34,45 +32,45 @@ const UploadPage: FunctionComponent = (): ReactElement => {
           } catch (ex) {
             setUploadingStatus(failed(ex.response.data));
           }
-        }}>
+        }}
+      >
         <input
           className="uploadForm__fileInput"
           type="file"
           ref={uploadInputRef}
-          accept=".xlsx,.xls" />
+          accept=".xlsx,.xls"
+        />
         <button type="submit">Загрузить</button>
       </form>
 
       <div className="uploadPageResult">
+        {isSucceeded(uploadingStatus) && (
+          <table className="uploadResultTable">
+            <thead>
+              <tr>
+                <th>Compound</th>
+                <th>Concentration</th>
+                <th>Signal</th>
+              </tr>
+            </thead>
 
-        {isSucceeded(uploadingStatus)
-      && (
-      <table className="uploadResultTable">
-        <thead>
-          <tr>
-            <th>Compound</th>
-            <th>Concentration</th>
-            <th>Signal</th>
-          </tr>
-        </thead>
+            <tbody>
+              {uploadingStatus.result.map(v => (
+                <tr key={uuidv4()}>
+                  <td>{v.compoundId}</td>
+                  <td>{v.concentration}</td>
+                  <td>{v.signal}</td>
+                </tr>
+              ))}
+            </tbody>
+          </table>
+        )}
 
-        <tbody>
-          {uploadingStatus.result.map((v) => (
-            <tr key={uuidv4()}>
-              <td>{v.compoundId}</td>
-              <td>{v.concentration}</td>
-              <td>{v.signal}</td>
-            </tr>
-          ))}
-        </tbody>
-      </table>
-      )}
+        {uploadingStatus.status === "loading" && <div className="spinner">Uploading...</div>}
 
-        {uploadingStatus.status === "loading"
-      && <div className="spinner">Uploading...</div>}
```

Comment:
why braces are needed? no, I can't tolerate it. let's remove it.

it's 100% convenient to write
```{statement &&
<Component>}```

TSX is really verbose, and syntax is really not always so easy readable, why do we need this one more thing which makes it's even harder?